src/test/java/uk/gov/hmcts/befta/player/DefaultBackEndFunctionalTestScenarioPlayerTest.java

```diff
@@ -332,6 +332,33 @@ public void shouldFailToVerifyThatThereIsAUserInTheContextWithAParticularSpecifi
         scenarioPlayer.verifyThatThereIsAUserInTheContextWithAParticularSpecification(specificationAboutUser);
     }
 
+    @Test
+    public void shouldVerifyThatThereIsASpecificationAboutScenarioContext() {
+        final String specificationAboutScenarioContext = "CONTEXT SPEC";
+        HttpTestData testData = mock(HttpTestData.class);
+
+        when(testData.meetsSpec(any())).thenReturn(true);
```

Comment:
this should ideally be tested against a value returned for an input of a specific spec. 
`when(testData.meetsSpec(any())).thenReturn(true);`
should be
`when(testData.meetsSpec(specificationAboutScenarioContext)).thenReturn(true);`